core/computed/metrics/lcp-load-start.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * @license Copyright 2023 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import {makeComputedArtifact} from '../computed-artifact.js';
+import {NavigationMetric} from './navigation-metric.js';
+import {LighthouseError} from '../../lib/lh-error.js';
+import {LargestContentfulPaint} from './largest-contentful-paint.js';
+import {ProcessedNavigation} from '../processed-navigation.js';
+import {TimeToFirstByte} from './time-to-first-byte.js';
+import {LCPRecord} from '../lcp-record.js';
+
+class LCPLoadStart extends NavigationMetric {
```

Comment:
Thinking more since last week, it does still seem like this will be simpler as a plain computed artifact instead of a `NavigationMetric`. The difference between observed and simulated isn't observed vs graph, it's the same value but scaled or not, which we handle in several other places without going through a full lantern metric. Dropping the `NavigationMetric` part would drop a lot of the setup required in here (jsdoc, split methods), but also the hopping up and down the unnecessary class hierarchy when following the execution flow or debugging.

Reply:
My biggest problems with that are:

- We would need to duplicate [the logic that decides which computation to use](https://github.com/GoogleChrome/lighthouse/blob/main/core/computed/metrics/metric.js#L84).
- Even though the simulated calculation is simple now, we might still update it in the future, possibly using a full lantern graph.

> The difference between observed and simulated isn't observed vs graph, it's the same value but scaled or not, which we handle in several other places without going through a full lantern metric

What other places are you talking about?

Reply:
Ok I tried it out, and your approach is 100% better. It even helped expose a bug in how we were simulating these values for text nodes that was increasing our error on load start and load end.